dm/pkg/parser/common.go

```diff
@@ -362,6 +362,10 @@ func genTableName(schema string, table string) *filter.Table {
 
 // CheckIsDDL checks input SQL whether is a valid DDL statement.
 func CheckIsDDL(sql string, p *parser.Parser) bool {
```

Comment:
for COMMIT we should treat it as a DDL to let caller save position information. Maybe this can be done at caller rather than in this function